### DISCUSSION TITLE
Plans to sync with upstream?

### DISCUSSION THREAD

**Discussion by catboxanon** [lllyasviel/stable-diffusion-webui-forge#80] [General]
Title. I hadn't seen mention of this anywhere in the README, and particularly after being invited as a collaborator (which I appreciate the thought!) I'm curious what the plan here is, if any. The other major fork, [SD.Next](https://github.com/vladmandic/automatic), originally intended to do this, but over time diverted, and is essentially it's own project now from my point of view. Several extension authors including myself gave up on support for it.

**Comment by lllyasviel** [comment]
We use bot to get commit from webui dev.

In forge, all python symbols and declarations are preserved even after their contents are cleaned.

For example 

https://github.com/lllyasviel/stable-diffusion-webui-forge/blob/main/modules/lowvram.py

compared to 

https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/master/modules/lowvram.py

So in fact this file is removed but all python symbols are still here because the file is not removed by me, it is removed by a bot with an algorithm over python symbols.

So in theory if my assumptions with python symbols are correct, it will be forever on latest webui dev with no maintaining cost.
